src/Routers/UsersRouter.js

```diff
@@ -289,6 +337,135 @@ export class UsersRouter extends ClassesRouter {
     }
     return Promise.resolve(success);
   }
+  encryptMfaKey(mfa, encryptionKey) {
+    try {
+      const algorithm = 'aes-256-gcm';
+      const encryption = crypto
+        .createHash('sha256')
+        .update(String(encryptionKey))
+        .digest('base64')
+        .substr(0, 32);
+      const iv = crypto.randomBytes(16);
+      const cipher = crypto.createCipheriv(algorithm, encryption, iv);
+      const encryptedResult = Buffer.concat([
+        cipher.update(mfa),
+        cipher.final(),
+        iv,
+        cipher.getAuthTag(),
+      ]);
+      return encryptedResult.toString('base64');
+    } catch (e) {
+      throw new Parse.Error(210, 'Invalid MFA token');
+    }
+  }
+  async decryptMFAKey(mfa, encryptionKey) {
+    try {
+      const algorithm = 'aes-256-gcm';
+      const encryption = crypto
+        .createHash('sha256')
+        .update(String(encryptionKey))
+        .digest('base64')
+        .substr(0, 32);
```

Comment:
avoid duplicated code with encryptMfaKey, may be you should create a separate little util function